options.go

```diff
@@ -339,6 +353,13 @@ func WithDisableMetadataExchange() Option {
 	}
 }
 
+// WithDisableBuiltInMetrics disables the built-in metrics for OpenTelemetry.
```

Comment:
You might want to expand on the description here.

Cf. https://github.com/GoogleCloudPlatform/alloydb-go-connector/blob/51b4f6a53ec2d23b508f8bc73d62050799135bae/options.go#L436-L441